code/game/objects/effects/landmarks.dm

```diff
@@ -21,26 +21,16 @@
 			delete_me = 1
 		if("JoinLate") // Bit difference, since we need the spawn point to move.
 			latejoin += src
-<<<<<<< HEAD
-			simulated = TRUE
-		//	delete_me = 1
-			return
-=======

```

Comment:
The diff in this file doesnt' quite show it, but there is one landmark (vinestart) in this file defined below where return has appeared. Not sure if it was fixed or not, just making sure.